### PR TITLE
update tikv docs for titan fallback

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

https://github.com/pingcap/docs/pull/10257

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10257
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: storage-engine/titan-configuration.md [hunk 1]
```diff
@@ -110,19 +110,41 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
...
+    - 执行 `tiup cluster edit-config`，编辑配置文件，再执行 `tiup cluster reload -R tikv`。
+    - 手动修改 TiKV 配置文件，然后重启 TiKV。
+
+    ```toml
+    [rocksdb.defaultcf.titan]
+    blob-run-mode = "fallback"
+    discardable-ratio = 1.0
+    ```
+
+2. 使用 tikv-ctl 执行全量 compaction。这一步骤将消耗大量 I/O 和 CPU 资源。
```
Comment (Oreoxmt) [inline storage-engine/titan-configuration.md:134]:
```suggestion
2. 使用 `tikv-ctl` 执行全量数据整理 (Compaction)。这一步骤将消耗大量 I/O 和 CPU 资源。
```

File: storage-engine/titan-configuration.md [hunk 2]
```diff
@@ -110,19 +110,41 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
...
+
+    ```bash
+    tikv-ctl --pd <PD_ADDR> compact-cluster --bottommost force
+    ```
+
+3. compaction 结束后，通过 **TiKV-Details**/**Titan - kv** 监控面板确认 **Blob file count** 指标降为 0。
```
Comment (Oreoxmt) [inline storage-engine/titan-configuration.md:140]:
```suggestion
3. 数据整理结束后，通过 **TiKV-Details**/**Titan - kv** 监控面板确认 **Blob file count** 指标降为 0。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -482,8 +482,8 @@ I/O rate limiter 相关的配置项。
 ### `mode`
 
 + 确定哪些类型的 I/O 操作被计数并受 `max-bytes-per-sec` 阈值的限流。当前 TiKV 只支持 write-only 只写模式。
-+ 可选值：write-only
-+ 默认值：write-only
++ 可选值：`"read-only"`，`"write-only"`，`"all-io"`
++ 默认值：`"all-io"`
```
Comment (tabokie) [inline tikv-configuration-file.md:486]:
```suggestion
+ 默认值：`"write-only"`
```

**Review by tabokie — APPROVED** [review]

**Pull request merged**